task4/src/main/scala/CsvTask.scala

```diff
@@ -0,0 +1,122 @@
+
+import java.io._
+import java.text.SimpleDateFormat
+
+import scala.language.reflectiveCalls
+import scala.collection.mutable.{ArrayBuffer, Map}
+
+import java.time.format.DateTimeFormatter
+
+
+
+object CsvTask extends  App {
+
+  using(io.Source.fromFile("201608-citibike-tripdata.csv")) { source =>
+
+  {
+    var alltrevel = 0
+    var allbikes = Set[String]()
+    var bike = Map[String, Int ]()
+    var Manth = Map[Int, Int]()
```

Comment:
1) variables should have camelCase naming
2) anyway, sounds like there's a grammar mistake